content/en/blog/falco-i18n.md

```diff
@@ -1,107 +1,104 @@
 
-```[languages.ko]
-title = "Falco"
-description = "런타임 보안"
-languageName = "한국어 Korean"
-weight = 3
-contentDir = "content/ko"
-languagedirection = "ltr"
+    ```[languages.ko]
+    title = "Falco"
+    description = "런타임 보안"
+    languageName = "한국어 Korean"
+    weight = 3
+    contentDir = "content/ko"
+    languagedirection = "ltr"
 
-[languages.ko.params]
-time_format_blog = "2006.01.02"
-language_alternatives = ["en"]
+    [languages.ko.params]
+    time_format_blog = "2006.01.02"
+    language_alternatives = ["en"]
 
-```
+    ```
 
-5. Create a directory, named  <lang code>, corresponding to your language in the [content](https://github.com/falcosecurity/falco-website/tree/master/content) directory.
+5. Create a directory, named  `<lang code>`, corresponding to your language in the [content](https://github.com/falcosecurity/falco-website/tree/master/content) directory.
 
    For example,  `content/ml` is the directory corresponding to the language, Malayalam.  
 
-6. Navigate to the directory. 
+6. Navigate to the language directory that you have just created.
 
    `cd content/<lang code>`
 
-   Replace <lang code> with your locale ID.
+   Replace `<lang code>` with your locale ID.
 
 7. Create a ` _index.md` file in the directory.
 
-   For example, see the [_index.md](https://github.com/falcosecurity/falco-website/blob/master/content/en/_index.md) file corresponding to English. 
+   For example, see the [_index.md](https://github.com/falcosecurity/falco-website/blob/master/content/en/_index.md) file corresponding to English.
 
-8. Translate the content to your language of choice.
+8. Translate the content to your language.
 
 9. Create a `content/<lang code>/docs` directory and a corresponding `_index.md` file in it.
 
-   Likewise, ensure that you create an`_index.md`  file for each directory and sub-directory in the `content/<lang code>`directory.
+   Likewise, ensure that you create an`_index.md`  file for each directory and sub-directory in the `content/<lang code>` directory.
 
-10. Start with the Getting Started section. Replicate the `getting-started` directory and file structure. 
+10. Start with the Getting Started section. Replicate the `getting-started` directory and its file structure.
 
-11. Translate the content in the Getting Started directory to your language of choice.
+11. Translate the content in the Getting Started directory to your language.
 
     For example, see the [English](https://github.com/falcosecurity/falco-website/tree/master/content/en/docs/getting-started) version of the Getting Started directory.
 
-12. Once ready create a PR against the master. Make sure you split the effort in smaller units.
+12. Ensure that you sign your commit with `git commit -s -m "message”`.
 
-13. Ensure that you sign your PR with `git commit -s -m "message”`
+13. Once ready create the PR against the master. Make sure you split the effort in smaller units.
 
 
-## Review Process
+## Become a Maintainer
```

Comment:
Perhaps, it worth mentioning https://github.com/falcosecurity/.github/blob/master/GOVERNANCE.md ?

Reply:
will add the link.